Orange/widgets/visualize/owheatmap.py

```diff
@@ -1314,15 +1311,21 @@ def update_grid_spacing(self):
             layout.setSpacing(self.SpaceX)
             self.__fixup_grid_layout()
 
-    def check_threshold_consistency(self):
-        return self.threshold_low < self.threshold_high
+    def update_lowslider(self):
+        if self.controls.threshold_low.value() == 20:
+            self.controls.threshold_low.setSliderPosition(19)
+        if self.controls.threshold_low.value() >= self.controls.threshold_high.value():
+            self.controls.threshold_high.setSliderPosition(self.controls.threshold_low.value() + 1)
+        self.update_color_schema()
```

Comment:
I don't like that if someone changes the slider parameters (range or steps) 800 lines higher, the hard-coded constants (20 and 19) here would need to be updated too (easy to miss).
How about avoiding the constants and making the code a bit shorter, with a function like:
```python
    def update_lowslider(self):
        low, high = self.controls.threshold_low, self.controls.threshold_high
        if low.value() >= high.value():
            low.setSliderPosition(high.value() - 1)
        self.update_color_schema()
```
And similarly for the highslider below.